tests/bucketlist_db_size_check.sql

```diff
@@ -0,0 +1,17 @@
+{{ config(
+    severity="error"
+    , tags=["singular_test"]
```

Comment:
FYI this tag runs every 30 mins in airflow. This is a higher frequency compared to what the cloud function and scheduled query tests used to run at. Which is good.

Just mentioning this in case we get noisy alerts where we might want to adjust the query and/or the frequency the tests are run (possibly with a separate dbt tag).